lib/avo/fields/files_field.rb

```diff
@@ -37,6 +40,14 @@ def fill_field(model, key, value, params)
 
         model
       end
+
+      def viewer_component
```

Comment:
What is the `viewer_compoent` and why does this live on the file field? Isn't this a "view layer" dependency?

We're passing params inside a file class to fetch the name of view component (outside-inside-outside). It seems a bit complex.

Let's move this somewhere closer to the view (maybe on a view component)?

Does this make sense?

Reply:
I'm not sure why I let that on the field, in some point of the iteration I was using that on several independent components, I noticed that now it's only used in one component, you're right, this do not belongs here, will fix on the next commit